course_catalog/views.py

```diff
@@ -419,3 +426,16 @@ def post(self, request):
         else:
             log.error("No records found in webhook: %s", request.body.decode())
         return Response({})
+
+
+class PodcastViewSet(viewsets.ReadOnlyModelViewSet):
+    """
+    Viewset for Podcasts
+    """
+
+    serializer_class = PodcastSerializer
+    permission_classes = (ReadOnly & PodcastFeatureFlag,)
+
+    queryset = Podcast.objects.filter(published=True).prefetch_related(
```

Comment:
Could you prefetch `offered_by` and `topics` on `Podcast` and `PodcastEpisode` to avoid N+1 queries?

Reply:
Yes, just committed the change